packages/elements/src/styles/vendor/pell.scss

```diff
@@ -14,6 +14,7 @@ $pell-content-padding: 10px !default;
   border-radius: 0;
   overflow: hidden;
   color: $black;
+  height: 180px;
```

Comment:
could we use `rem` instead of `px` ?

Reply:
No, because the height must be equal with the existing summary field which has `180px` as height